packages/daimo-userop/src/index.ts

```diff
@@ -222,11 +213,7 @@ export class DaimoOpSender {
     return this.sendUserOp(op);
   }
 
-  /**
-   * Creates an ephemeral note with given value.
-   * Infinite-approves the notes contract first, if necessary.
```

Comment:
Fixes Veridise Audit ID 11

Reply:
i recommend extracting that into its own PR

Reply:
done; #276